package.xml

```diff
@@ -16,11 +16,13 @@
   <author>Felix Messmer</author>
   <author>Miguel Prada Sarasola</author>
   <author>Denis Stogl</author>
+  <author>Andy Zelenak</author>
+  <author>Marvin Grosse Besselmann</author>
 
-  <maintainer email="grosse@fzi.de">Marvin Große Besselmann</maintainer>
+  <maintainer email="exner@fzi.de">Felix Exner</maintainer>
+  <maintainer email="schnell@fzi.de">Tristan Schnell</maintainer>
   <maintainer email="lovro.ivanov@gmail.com">Lovro Ivanov</maintainer>
   <maintainer email="denis.stogl@picknik.ai">Denis Stogl</maintainer>
```

Comment:
I am still planing to contribute to the package since I am using it a lot in other projects. And we have some fun scenario there, so I expect to do some small tweaks in the future, e.g., better multi-robot support for simulations (short-term).

```suggestion
  <maintainer email="denis@stoglrobotics.de">Denis Stogl</maintainer>
```